_posts/2019-02-07-jan-updates.md

```diff
@@ -0,0 +1,51 @@
+---
+title: January Updates - Celebrate 'I Love Free Software Day' 
+author: Hong Phuc Dang 
+date: 2019-02-07
+banner:
+  image: jan-banner.png
+---
+### This is a recap of open source activities and development at Zalando in the month of January
+
+## Project Highlights 
```

Comment:
I think the harassment policy blogpost should also be added to the january review